### DISCUSSION TITLE
dual pistols

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#5620] [Ideas and suggestions]
Add a unique weapon, two pistols, as a two handed (single slot) weapon.

Or perhaps like the revolver, but two of them, and have [twelve shots.](https://cdn.discordapp.com/attachments/220881718666723328/836260580880678952/Twelve_shots.mp4)

reason: It would look really really cool

**Comment by Rokvach** [comment]
You can already have dual revolvers, just equip one for each hand.
